src/main/java/com/microsoft/azure/management/cosmosdb/samples/CreateCosmosDBWithEventualConsistency.java

```diff
@@ -106,34 +106,28 @@ public static boolean runSample(Azure azure, String clientId) {
         return false;
     }
 
-    private static void createDBAndAddCollection(String masterKey, String endPoint) throws DocumentClientException {
+    private static void createDBAndAddCollection(String masterKey, String endPoint) throws CosmosException {
         try {
-            DocumentClient documentClient = new DocumentClient(endPoint,
-                    masterKey, ConnectionPolicy.GetDefault(),
-                    ConsistencyLevel.Session);
+            CosmosClient cosmosClient = new CosmosClientBuilder()
+                .endpoint(endPoint)
+                .key(masterKey)
+                .consistencyLevel(ConsistencyLevel.SESSION).buildClient();
 
-            // Define a new database using the id above.
-            Database myDatabase = new Database();
-            myDatabase.setId(DATABASE_ID);
 
-            myDatabase = documentClient.createDatabase(myDatabase, null)
-                    .getResource();
+            cosmosClient.createDatabase(DATABASE_ID);
 
-            System.out.println("Created a new database:");
-            System.out.println(myDatabase.toString());
+            CosmosDatabase cosmosDatabase = cosmosClient.getDatabase(DATABASE_ID);
 
-            // Define a new collection using the id above.
-            DocumentCollection myCollection = new DocumentCollection();
-            myCollection.setId(COLLECTION_ID);
+            System.out.println("Created a new database:");
+            System.out.println(cosmosDatabase.toString());
```

Comment:
I could not find a `toString` method in `CosmosDatabase` type, so not sure this will print any useful info other than the class name.